tests/integration/mock_vms_server.py

```
@@ -0,0 +1,348 @@
+            if resource_id not in _STORE[resource]:
+                found = False
+            else:
+                found = True
+                del _STORE[resource][resource_id]
+        if not found:
+            self._respond(404, {"error": f"{resource} {resource_id} not found"})
+        else:
+            self.send_response(204)
+            self.end_headers()
+
+    def log_message(self, format, *args):
```

Ruff (0.15.14)

[error] Function argument `format` is shadowing a Python builtin

(A002)